frontend/assets/l10n/nuernberg/overwrite_de.json

```diff
@@ -0,0 +1,5 @@
+{
+  "about": {
+    "title": "Nürnberg"
```

Comment:
is this just for testing purposes? i think this tab title makes no sense

Reply:
Yes its for testing purposes only, will be fixed in #1189.